docs/user-guides/operations.md

```diff
@@ -134,9 +134,9 @@ The operator will automatically detect the changes and update the deployed Vald
 
 ### References
 
-- [vald-helm-chart](vald-helm-chart)
-- [vald-helm-operator-chart](vald-helm-operator-chart)
-- [CHANGELOG](CHANGELOG)
```

Comment:
Could you remove these definition?
```
[vald-helm-chart]: https://github.com/vdaas/vald/tree/main/charts/vald
[vald-helm-operator-chart]: https://github.com/vdaas/vald/tree/main/charts/vald-helm-operator
[CHANGELOG]: https://vald.vdaas.org/docs/release/changelog/
```

Reply:
vald-helm-chart and CHANGELOG is used in other place, so I will keep these 2 links here

Reply:
Should we change all?

Reply:
the link is shared so I think we can define here

Reply:
sorry, again

Reply:
fixed